common/test/unit/com/thoughtworks/go/domain/ArtifactPlanTest.java

```diff
@@ -139,7 +140,7 @@ public void validate_shouldFailIfSourceIsEmpty() {
    public void validate_shouldFailIfDestDoesNotMatchAFilePattern() {
         ArtifactPlan artifactPlan = new ArtifactPlan("foo/bar", "..");
         artifactPlan.validate(null);
-        assertThat(artifactPlan.errors().on(ArtifactPlan.DEST), is("Invalid destination path. Destination path should match the pattern ([^. ].+[^. ])|([^. ][^. ])|([^. ])"));
+        assertThat(artifactPlan.errors().on(ArtifactPlan.DEST), is("Invalid destination path. Destination path should match the pattern " + FilePathTypeValidator.PATH_PATTERN));
```

Comment:
Better error message please :) Something along the lines of "must be a relative path" reads better than a regular expression.